.env.local

```
@@ -1,20 +1,32 @@
-STORAGE_DRIVER=minio
-MINIO_SECURE=false
-S3_ACCESS_KEY=MdnaDEXKy9nOc4beIvNcgy
-S3_SECRET_KEY=BjjctVsoSg4FKkT81VKt18
-S3_BUCKET_NAME_PRIVATE=cb-faynosync-s3
+STORAGE_DRIVER=aws
+S3_ACCESS_KEY=GK8fcbbed327a9ff97f250eb4e
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The S3_ACCESS_KEY key should go before the STORAGE_DRIVER key

(UnorderedKey)